sdk/loadtesting/azure-developer-loadtesting/generated_tests/test_load_test_run_async.py

```diff
@@ -13,217 +13,6 @@
 
 @pytest.mark.skip("you may need to update the auto-generated test case before run it")
 class TestLoadTestRunAsync(LoadTestRunClientTestBaseAsync):
-    @LoadTestRunPreparer()
-    @recorded_by_proxy_async
-    async def test_begin_test_run(self, loadtestrun_endpoint):
-        client = self.create_async_client(endpoint=loadtestrun_endpoint)
-        response = await client.begin_test_run(
```

Comment:
Are there other tests covering all the functionality that's being checked in all of these removed tests?